### PR TITLE
The date problem in ran_at field was fixed and the parameter field type was changed to text and the readme was completed

### PR DESCRIPTION
**Pull request by mohsen-farahani**
The date problem in ran_at field was fixed and the parameter field type was changed to text and the readme was completed

### CODE REVIEW

**Comment by roshangautam** [comment]
can you please fix the failing tests and also style ci issues ?

File: README.md [hunk 1]
```diff
@@ -152,6 +152,11 @@ In addition to the dashboard, Totem provides an artisan command to view a list o
 php artisan schedule:list
 ```
 
+If you find that your changes do not apply, run the command below to empty and rebuild the cache
+```
+php artisan queue:work --queue=queue_default
```
Comment (roshangautam) [inline README.md:157]: Can you please help me understand how will this rebuild the cache?

File: database/migrations/2019_08_26_154840_alter_task_results_table_change_type_ran_at_fileds.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterTaskResultsTableChangeTypeRanAtFileds extends Migration
```
Comment (roshangautam) [inline database/migrations/2019_08_26_154840_alter_task_results_table_change_type_ran_at_fileds.php]: nit: typo in `Fields`

**Review by roshangautam — CHANGES_REQUESTED** [review]
Is there an open issue for this pull request? If not can you please add the problem statement in your commit description? I am trying to understand what's broken and how is it being fixed?

**Comment by roshangautam** [comment]
any updates here @mohsen-farahani ?

File: README.md (hunk 1 above)
Comment (mohsen-farahani) [reply README.md:157]: The cache needs to be cleared when any changes are made.
And this works when the queues are running.
I would have liked to discuss this better here.

**Comment by roshangautam** [comment]
Can you please update the description with the problem statement? I would really appreciate if you can help me understand the problem you are trying to solve with this PR. Thanks.

**Comment by mohsen-farahani** [comment]
> Can you please update the description with the problem statement? I would really appreciate if you can help me understand the problem you are trying to solve with this PR. Thanks.

when we change any tasks configs (for example disable one task) we should update cache manually with
`php artisan cache:clear`
or run queue worker that do it for us
Totem dispatch a job for each change to
`default_queue‍‍‍`
Any where in Readme don't say this and may make misunderstanding

**Pull request closed without merging**